packages/adblocker/src/filters/network.ts

```diff
@@ -451,6 +451,83 @@ function getFilterOptionValue(line: string, pos: number, end: number): [number,
   return [pos, value];
 }
 
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_escape
+ * \f, \n, \r, \t, \v, \0, \^, \$, \\, \., \*, \+, \?, \(, \), \[, \], \{, \}, \|, \/
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape
+ * \d, \D, \s, \S, \w, \W
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Word_boundary_assertion
+ * \b, \B
+ */
+const REGEXP_CHARACTER_ESCAPES = new Set([
+  102, 110, 114, 116, 118, 48, 94, 36, 92, 46, 42, 43, 63, 40, 41, 91, 93, 123, 125, 124, 47, 100,
+  68, 119, 87, 115, 83, 98, 66,
+]);
+
+function isHexLiteral(code: number) {
+  return (code >= 48 && code <= 57) || (code <= 65 && code <= 70) || (code >= 97 && code <= 102);
+}
+
+/**
+ * Checks if the character(s) after the escape character should be preserved.
+ */
+function isCharacterEscapeInRegExp(line: string, pos: number, _end: number): [number, boolean] {
+  const code = line.charCodeAt(pos + 1);
+  if (REGEXP_CHARACTER_ESCAPES.has(code) || code === 44 /* ',' */) {
+    return [pos + 1, true];
+  }
+  // \cA, \cB, …, \cz
+  if (code === 99 /* 'c' */) {
+    const next = line.charCodeAt(pos + 2);
+    if ((next >= 65 && next <= 90) || (next >= 97 && next <= 122)) {
+      return [pos + 2, true];
+    }
+  }
+  // \xHH
+  if (
+    code === 120 /* 'x' */ &&
+    isHexLiteral(line.charCodeAt(pos + 2)) &&
+    isHexLiteral(line.charCodeAt(pos + 3))
+  ) {
+    return [pos + 3, true];
+  }
+  if (code === 117 /* 'u' */) {
+    // \u{HHH}
+    if (line.charCodeAt(pos + 2) === 123 /* '{' */) {
+      const close = line.indexOf('}', pos + 3);
+      const hexLiteralLength = close - pos + 3;
+      if (hexLiteralLength >= 1 && hexLiteralLength <= 6) {
+        return [close, true];
+      }
+    } else if (
+      // \uHHHH
+      isHexLiteral(line.charCodeAt(pos + 2)) &&
+      isHexLiteral(line.charCodeAt(pos + 3)) &&
+      isHexLiteral(line.charCodeAt(pos + 4)) &&
+      isHexLiteral(line.charCodeAt(pos + 5))
+    ) {
+      return [pos + 5, true];
+    }
+  }
+  return [pos + 1, false];
+}
+
+/**
+ * https://adguard.com/kb/general/ad-filtering/create-own-filters/#replace-modifier
+ * \,, \/
```

Comment:
That comment confused me a bit. The set contains `,` and `/`.

Maybe we can make it clearer? What should `\,,` and `\/` mean here? If it is just `,` and `/`, then I would recommend to not quote it in the comment.

(Detail: if the Set will be that small, we could also just use an array.)

Reply:
Fixed in [`62acdc5` (#4241)](https://github.com/ghostery/adblocker/pull/4241/commits/62acdc551b5ec473401b1a1eb16064133cdcf7d8)